<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotViewer</class>
 <widget class="QWidget" name="PlotViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1382</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCustomPlot" name="widget_plot" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="widget_plot_2" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="widget_plot_3" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>设备1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_connect0">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_type0">
             <property name="minimumSize">
              <size>
               <width>86</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>86</width>
               <height>25</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>主雷达</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>副雷达</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_save0">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>设备2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_connect1">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>连接</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_type1">
             <property name="minimumSize">
              <size>
               <width>86</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>86</width>
               <height>25</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>主雷达</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>副雷达</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_save1">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>保存</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
